08-components-lab/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,49 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class TimeConverter extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+            number : 0
+        }
+    }
+
+    static propTypes = {
+        number: React.propTypes.number.isRequired
+    };
+
+    getSeconds(){
+        return Number(this.state.number);
+    };
+    getMinutes(){
+        return Number(this.state.number / 60);
+    };
+    getHours(){
+        return Number(this.state.number / 3600);
+    }
+
+    calc = (evt) => {
+        //reset numbers before setting new val
+        this.setState({number:0});
```

Comment:
When you call setState you trigger rerender. 
Any particular reason to reset numbers before setting new value?